content/outcomes/application-observability/tsm-posture-health-perf-distributed-app-day1.md

```diff
@@ -0,0 +1,38 @@
+---
+date: '2021-07-13'
+lastmod: '2021-07-13'
+layout: single
+team:
+  - Mark Schweighardt
+  - Niran Even-Chen
+  - Andrew Huffman
+title: "Tanzu Service Mesh: Understand the posture (health and performance) of your distributed application (day 1)"
+weight: 101
+---
+
+In very simple terms, a "modern application" is one that the functions and components are broken into a separate set of micro-services. Each function is considered a service, which is an app on its own, where together they create the "Application". These micro-services communicate with each other over the network and because they can be placed on different infrastructures and clouds, it requires the organization to be able to understand the application posture so that it is easier and faster to figure out what is the problem in case that issues arise. 
+
+The following views in Tanzu Service Mesh help the application owner get insight into how the application is built. Each view can be set in 5 minute, 15 , 30 and 1 hour data sets. This is configured in the top right hand of the screen
```

Comment:
This sentence doesn't read quite right to me, and there's some extra whitespace around the comma separating 15 and 30 minutes.

Reply:
took a crack at fixing this